sktime/forecasting/ets.py

```diff
@@ -242,7 +242,7 @@ def __init__(
         
         if self.auto:
             # If auto=True, check if trend, damped_trend, seasonal, or error have been passed in by user
-            if any([trend, damped_trend, seasonal, error]):
+            if any(param is not None for param in [trend, damped_trend, seasonal, error]):
```

Comment:
Accounts for user passing in False for certain params.

Reply:
I disagree with this now, as this will also raise a warning for some default settings - this would confuse the user.

Reply:
ah, yes, that makes sense.